doc/source/whatsnew/v1.3.0.rst

```diff
@@ -166,6 +166,56 @@ a copy will no longer be made (:issue:`32960`)
 The default behavior when not passing ``copy`` will remain unchanged, i.e.
 a copy will be made.
 
+.. _whatsnew_130.arrow_string:
+
+PyArrow backed string data type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We've enhanced the :class:`StringDtype`, an extension type dedicated to string data.
+(:issue:`39908`)
+
+It is now possible to specify a ``storage`` keyword option to :class:`StringDtype`, use
```

Comment:
what happens if someone constructs with `dtype='string'` e.g. are we deprecating, erroring, or defaulting to `string[python]`? this is experimental after all

Reply:
```
>>> pd.Series(list("abc"), dtype="string")
0    a
1    b
2    c
dtype: string[python]
>>> 
>>> with pd.option_context("string_storage", "pyarrow"):
...     print(pd.Series(list("abc"), dtype="string"))
... 
0    a
1    b
2    c
dtype: string[pyarrow]
>>> 
```

Reply:
added to release note